README.md

```diff
@@ -28,6 +28,6 @@ One of the dangers with an auto-provisioning course approach is the proliferatio
     
    
 ## Run Directions
-1. `java -jar canvasCourseDeleteUtility.jar file:/<path>/canvasCourseDelete.properties`
+1. `java -Dlog4j.configuration=file:/<path-to-log4j-file>/log4j.properties -jar canvasCourseDeleteUtility.jar file:/<path>/canvasCourseDelete.properties`
```

Comment:
Consider creating a run.sh command file which could both document the exact details required to start the program but also be a working example of how to do it.